# MeasEpochChannelType1 block
   
uint8  rx_channel
uint8  type
uint8  sv_id
uint8  misc
uint32 code_lsb
int32  doppler
uint16 carrier_lsb
int8   carrier_msb
uint8  cn0
uint16 lock_time
uint8  obs_info
uint8  n2

MeasEpochChannelType2[] type2